src/middlewared/middlewared/plugins/smb.py

```diff
@@ -1639,7 +1644,10 @@ async def setacl(self, data):
             })
         return await self.getacl({'share_name': data['share_name']})
 
-    @accepts(Dict('smb_getacl', Str('share_name', required=True)), roles=['SHARING_SMB_READ'])
+    @accepts(Dict(
+        'smb_getacl',
+        Str('share_name', required=True)
+    ), roles=['SHARING_SMB_READ'], audit='Setacl SMB share', audit_extended=lambda data: data['share_name'])
```

Comment:
'Setacl SMB share'... typo.
Also, will auditing the 'get' potentially cause a great deal of audit entries?

Reply:
It's unfortunately required in this case.